client/src/components/settings/toggle-setting.css

```
@@ -78,52 +82,49 @@
   cursor: default;
 }
 
-.toggle-radio-group {
-  display: flex;
-  align-items: start;
-}
-
-.toggle-radio-group label {
-  font-weight: normal;
-  display: flex;
-  align-items: center;
+.toggle-radio-group label + label {
+  margin-inline-start: 2rem;
 }
 
 .toggle-radio-group input {
-  -webkit-appearance: none;
-  appearance: none;
-  height: 0.8rem;
-  width: 0.8rem;
-  border-radius: 50%;
-  background: transparent;
-  border: 2px solid var(--secondary-color);
-  margin: 0;
-}
-
-.toggle-radio-group [data-checked='true'] input {
-  background: var(--secondary-color);
-}
-
-.toggle-radio-group [data-checked='true'] span {
-  font-weight: 700;
+  position: absolute;
+  left: -9999px;
 }
 
-.toggle-radio-group input:focus-visible {
-  outline-offset: 1px;
+.custom-circle {
+  margin-inline-end: 8px;
+  position: relative;
+  top: 2px;
+  display: inline-block;
+  min-width: 20px;
+  min-height: 20px;
+  max-width: 20px;
+  max-height: 20px;
```

```suggestion
  min-width: calc(0.889em + 4px);
  min-height: calc(0.889em + 4px);
  max-width: calc(0.889em + 4px);
  max-height: calc(0.889em + 4px);
```

This gives us the same dimensions but allows for more flexibility with users who manually adjust their font size.